### ISSUE TITLE
Scales and white patches on resultant picture

### ISSUE THREAD

**Issue by Vaibhav-nn** [MathiasGruber/PConv-Keras#62]
![S1](https://user-images.githubusercontent.com/42914799/82458986-03c25380-9ad5-11ea-9b6a-37440057008d.jpg)

There are white patches on my final image (right). Please, anyone, figure out my mistake.

**Comment by ceres99** [comment]
I ran into the same problem. I trained the model for 75 epochs with batch normalization in the first phase, then for 25 epochs freezing the batch normalization. I noticed that the white patches appear only in the second phase. 
Additional info:
- size of dataset: 4800 images
- batch size = 4
- steps per epoch = 1000
- validation steps = 100

![Screenshot (23)](https://user-images.githubusercontent.com/74055962/122438636-96b51200-cfa3-11eb-85e8-7db42b1fbaf7.png)

Any suggestions about what may cause the problem?
